src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemv/start-debian-template

```diff
@@ -22,47 +22,45 @@ export JAVA_OPTS
 # If program manages its own PID file then it
 # should declare its location in PIDFILE
 if [ -z "$PIDFILE" ]; then
-    create_pidfile=true
-    PIDFILE=/var/run/${{app_name}}/running.pid
+  create_pidfile=true
+  PIDFILE=/var/run/${{app_name}}/running.pid
 fi
 
 if [ -z "$DAEMON_USER" ]; then
-    DAEMON_USER=${{daemon_user}}
+  DAEMON_USER=${{daemon_user}}
 fi
 
 if [ -z "$DAEMON_GROUP" ]; then
-    DAEMON_GROUP=${{daemon_group}}
+  DAEMON_GROUP=${{daemon_group}}
 fi
 
-
 RUN_CMD="${{chdir}}/bin/${{exec}}"
 
-
 start_daemon() {
-    log_daemon_msg "Starting" "${{app_name}}"
-    [ -d "/var/run/${{app_name}}" ] || install -d -o "$DAEMON_USER" -g "$DAEMON_GROUP" -m755 "/var/run/${{app_name}}"
-    logfile="${{daemon_log_file}}"
-    stdout_redirect=""
-    if [ ! -z "${logfile:-}" ]; then
-      stdout_redirect=" >> ${{logdir}}/${{app_name}}/$logfile 2>&1"
-    fi
-
-    if [ "$create_pidfile" = true ]; then
-        start-stop-daemon --background --chdir ${{chdir}} --chuid "$DAEMON_USER" --make-pidfile --pidfile "$PIDFILE" --startas "$RUN_CMD" --start -- $RUN_OPTS "$stdout_redirect"
-    else
-        start-stop-daemon --background --chdir ${{chdir}} --chuid "$DAEMON_USER" --pidfile "$PIDFILE" --startas "$RUN_CMD" --start -- $RUN_OPTS "$stdout_redirect"
-    fi
-    log_end_msg $?
+  log_daemon_msg "Starting" "${{app_name}}"
+  [ -d "/var/run/${{app_name}}" ] || install -d -o "$DAEMON_USER" -g "$DAEMON_GROUP" -m755 "/var/run/${{app_name}}"
+  logfile="${{daemon_log_file}}"
+  stdout_redirect=""
+  if [ ! -z "${logfile:-}" ]; then
+    stdout_redirect=" >> ${{logdir}}/${{app_name}}/$logfile 2>&1"
+  fi
+
+  if [ "$create_pidfile" = true ]; then
+    start-stop-daemon --background --chdir ${{chdir}} --chuid "$DAEMON_USER" --make-pidfile --pidfile "$PIDFILE" --startas "$RUN_CMD" --start -- $RUN_OPTS ${stdout_redirect}
+  else
+    start-stop-daemon --background --chdir ${{chdir}} --chuid "$DAEMON_USER" --pidfile "$PIDFILE" --startas "$RUN_CMD" --start -- $RUN_OPTS ${stdout_redirect}
+  fi
+  log_end_msg $?
```

Comment:
It's hard to see the actual changes with all the formatting changes. Can you remove the formatting?